### ISSUE TITLE
feature: `kpm clean` to clean the package cache

### ISSUE THREAD

**Issue by zong-zhe** [kcl-lang/kpm#315]
## Feature Request

**Is your feature request related to a problem? Please describe:**

The process of downloading dependencies by kpm often depends on the network. In bad network, an incomplete dependency can sometimes cause unexpected things to happen, so we need to provide a command `kpm clean` to help users clean up these incomplete dependencies.

**Describe the feature you'd like:**

`kpm clean` to help user remove the depenendencies in local file system.

**Describe alternatives you've considered:**

It is also possible to let the user directly to the kpm cache directory `$HOME/.kcl/kpm` to delete it manually, but this can lead to accidents.

**Teachability, Documentation, Adoption, Migration Strategy:**

**Comment by officialasishkumar** [comment]
Can you please assign me this issue?

**Comment by officialasishkumar** [comment]
Does the hash gets changed during incomplete dependency download? Also any way to replicate this behaviour in local machine?

**Comment by Peefy** [comment]
> Does the hash gets changed during incomplete dependency download? Also any way to replicate this behaviour in local machine?

No. You can delete/destroy any package in the `$HOME/.kcl/kpm` path.

**Comment by literalEval** [comment]
Hey @Peefy 
I would like to take up this issue if it is not being actively worked on.
